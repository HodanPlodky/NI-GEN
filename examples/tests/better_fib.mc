int fib(int n) {
    int a = 0;
    int b = 1;
    for (int i = 0; i < n; i = i + 1) {
        int tmp = b;
        b = a + b;
        a = tmp;
    }
    return a;
}

int main() {
    return fib(40);
}
