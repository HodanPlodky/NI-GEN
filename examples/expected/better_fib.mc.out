input file : examples/tests/better_fib.mc
return 203
